templates/common/baremetal/files/NetworkManager-static-dhcp.yaml

```diff
@@ -0,0 +1,30 @@
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/30-static-dhcp"
+contents:
+  inline: |
+    #!/bin/bash
+    set -ex
+
+    if [ ${DHCP4_DHCP_LEASE_TIME:-0} -lt 4294967295 ]
```

Comment:
Does this not exist in IPv6?

Reply:
I only implemented ipv4 in this version. Based on my discussion with the NM team, I would probably argue that for this use case in ipv6 we should just use SLAAC addressing. If there's a reason we can't do that then we can always add support for DHCPv6 here too.